parlai/core/torch_ranker_agent.py

```diff
@@ -52,6 +52,11 @@ def add_cmdline_args(cls, argparser):
             choices=['batch', 'inline', 'fixed', 'vocab', 'batch-all-cands'],
             help='The source of candidates during evaluation (defaults to the same'
                  'value as --candidates if no flag is given)')
+        agent.add_argument(
+            '--repeat_blocking_heuristic', type='bool', default=True,
```

Comment:
i think the convention we're using here is `-` instead of `_`

Reply:
well if the convention was '_' everywhere because then it's easier to search in a file to find the same thing...  alex made it so that it converts to '-' in the actual flags..

Reply:
so my suggestion is to change the convention..

Reply:
you can just grep for repeat.blocking.heuristic to find them both if that's really what you want. 

```bash
$ ag 'add_argument.*--[a-z]+-' | wc -l
383
```

I think the convention is a bit stronger than one preference here...